neurodamus/connection.py

```diff
@@ -527,11 +529,11 @@ def _create_synapse(self, cell, params_obj, x, syn_id, base_seed):
         """
         is_inh = params_obj["synType"] < 100
         if self._mod_override is not None:
-            mod_override = self._mod_override.get("ModOverride").s
+            mod_override = self._mod_override
```

Comment:
It seems that the var `mod_override` is not needed, we can pass directly `self._mod_override` to build `override_helper` ?

Reply:
ah yes, that is a relic of debugging